ios/MaasCore/.gitignore

```diff
@@ -0,0 +1,5 @@
+.DS_Store
+/.build
+/Packages
+/*.xcodeproj
+xcuserdata/
```

Comment:
Since our repository root `.gitignore` file already includes all of these this additional file may not be necessary.

Reply:
These belongs to MaasCore Package 📦. I say we should keep this there.